include/audio_backend/__coreaudio_backend.h

```diff
@@ -137,11 +137,12 @@ class audio_device {
     return sample_rate;
   }
 
-  span<const sample_rate_t> get_supported_sample_rates() const noexcept {
+  // TODO: remove tcb namespace when std::span is used
```

Comment:
Rather than referring to `tcb::span` directly, take a look at my fix from the `wasapi` branch, which replaces the `using namespace` declaration with a templated `using` declaration for the `span` class, which allows it to be spelled as `span` in the various backends, while still allowing the Windows headers to be included.

Reply:
Fixed in b3fdb8f